prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -271,3 +273,730 @@ periodics:
       - make
       - test-e2e-gke-multi-repo-test-group2
       - 'GCP_CLUSTER=multi-repo-2-autopilot-rapid-latest'
+
+### multi-repo test group 3 jobs ###
+- <<: *config-sync-ci-job
+  name: multi-repo-3-standard-stable
+#  TODO: enable the annotations for the testgrid dashboard when we get all prow jobs pass.
```

Comment:
unfortunately I don't think the presubmit lets you add periodic jobs without adding them to a testgrid

Reply:
That's why the validation failed. Okay. I'll add the testgrid configs then. Thanks!

Reply:
DONE